### PR TITLE
Rename search_clients_daily

### PR DESCRIPTION
**Pull request by tdsmith**

### CODE REVIEW

File: src/mozanalysis/metrics/desktop.py [hunk 1]
```diff
@@ -11,18 +11,22 @@
...
-#: .. _`search_clients_daily`: https://docs.telemetry.mozilla.org/datasets/
-#:    search/search_clients_daily/reference.html
-search_clients_daily = DataSource(
-    name="search_clients_daily",
-    from_expr="`moz-fx-data-shared-prod.search.search_clients_daily`",
+#: .. _`search_clients_engines_sources_daily`: https://docs.telemetry.mozilla.org/
+#:    datasets/search/search_clients_engines_sources_daily/reference.html
+search_clients_engines_sources_daily = DataSource(
+    name="search_clients_engines_sources_daily",
+    from_expr="`moz-fx-data-shared-prod.search.search_clients_engines_sources_daily`",
```
Comment (jklukas) [inline src/mozanalysis/metrics/desktop.py:22]: Note that all user-facing views are published in both shared-prod and mozdata. Preferred style today would be to use mozdata and drop the backticks here.

Comment (tdsmith) [reply src/mozanalysis/metrics/desktop.py]: Thanks for the callout!

File: src/mozanalysis/metrics/desktop.py [hunk 2]
```diff
@@ -61,7 +65,7 @@
                     *,
                     DATE(submission_timestamp) AS submission_date,
                     environment.experiments
-                FROM `moz-fx-data-shared-prod`.telemetry_stable.main_v4
+                FROM mozdata.telemetry_stable.main_v4
```
Comment (scholtzan) [inline src/mozanalysis/metrics/desktop.py:68]: This needs to be `moz-fx-data-shared-prod`. `mozdata` doesn't have access to stable or derived tables

```suggestion
                FROM `moz-fx-data-shared-prod`.telemetry_stable.main_v4
```

**Review by scholtzan — APPROVED** [review]
r+wc

File: src/mozanalysis/metrics/desktop.py (hunk 2 above)
Comment (tdsmith) [reply src/mozanalysis/metrics/desktop.py]: thanks for the catch!

**Pull request merged**